core/src/components/cat-avatar/cat-avatar.scss

```diff
@@ -43,6 +43,7 @@ $-sizes: (
   white-space: nowrap;
   overflow: hidden;
   vertical-align: middle;
+  font-weight: 600;
```

Comment:
It is better to modify the font-weight from the style sheet of the select, since we only want to modify the initials inside the select, right?

Reply:
Why do we want to change the font weight in the first place? The avatar should be an independent component and should not be influenced by this. What is the reasoning @mrkskmn ?

Reply:
I am missing context. 
Is this about https://haiilo.atlassian.net/browse/PMI-819? 
If so, I was just handing off the text styles from Figma. Maybe there was a mismatch between the code and Figma if they haven't had the font-weight:600 already.

Reply:
ok, then this works 👍